classes/security/authorization/internal/SectionAssignmentRule.inc.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @file classes/security/authorization/internal/SectionAssignmentRule.inc.php
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2000-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SectionAssignmentRule
+ * @ingroup security_authorization_internal
+ *
+ * @brief Class to check if there is an assignment
+ * between user and a section/series.
```

Comment:
I think that as far as we are using Section as a generic name for both section and series, we can avoid using series anywhere else in library...